### PR TITLE
New config test cases

### PR DESCRIPTION
**Pull request by fruch**

### CODE REVIEW

File: sdcm/sct_config.py [hunk 1]
```diff
@@ -559,6 +559,13 @@ class SCTConfiguration(dict):
         dict(name="pre_create_schema", env="SCT_PRE_CREATE_SCHEMA", default=None, type=boolean, required=False,
              help=""),
 
+        dict(name="validate_partitions", env="SCT_VALIDATE_PARTITIONS", default=None, type=boolean, required=False,
+             help=""),
```
Comment (bentsi) [inline sdcm/sct_config.py]: please add help? what are those used for?

**Comment by fruch** [comment]
@roydahan can we sit and review those use cases ?

**Comment by roydahan** [comment]
Yes, let's review this together.

**Comment by bentsi** [comment]
@roydahan are we ready to merge this?

**Comment by roydahan** [comment]
I need to review new commits from @fruch.
@fruch did you go over all params.get that we have in the code, or just what we currently have in the tests yamls?

**Comment by fruch** [comment]
@roydahan I went over the whole code, but you know I'm still human, and might missed some (once working on those new test cases, I've found a few things I've missed or got wrong)

We won't know until we won't run some of those jobs, for a few rounds.

File: defaults/test_default.yaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
+db_type: scylla
+
+ip_ssh_connections: false
```
Comment (fruch) [inline defaults/test_default.yaml]: change to 'private'

File: defaults/test_default.yaml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+# for for version selection
+scylla_linux_distro: 'centos'
+
+monitor_branch: 'branch-2.2'
+store_results_in_elasticsearch: true
+
+
+reuse_cluster: false
+nemesis_class_name: 'NoOpMonkey'
+nemesis_during_prepare: false
```
Comment (fruch) [inline defaults/test_default.yaml]: should be 'true' (as string)

File: test-cases/gemini/gemini-basic-3h.yaml [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+# gemini
+# cmd: gemini -d -n [NUM_OF_TEST_ITERATIONS] -c [NUM_OF_THREADS] -p [NUM_OF_PARTITION_KEYS_PER_THREAD] -m mixed -f
+# the below cmd runs about 3 hours
+gemini_cmd: "gemini -d -n 500000 -c 20 -p 100 -m mixed -f"
+gemini_url: 'https://s3.amazonaws.com/scylla-gemini/Binaries/gemini-latest'
+gemini_static_url: 'https://s3.amazonaws.com/scylla-gemini/Binaries/schema.json'
+
+db_type: mixed_scylla
+instance_type_db_oracle: 'i3.8xlarge'
+ami_id_db_oracle: 'ami-0226296be51b08fa4'
```
Comment (fruch) [inline test-cases/gemini/gemini-basic-3h.yaml]: change to the official of 2.3, and add the remark

**Comment by fruch** [comment]
FYI
@amoskong @aleksbykov,
in master we are switching soon to use those new configuration,
new PR should start using those options
this PR would be merged today.
approved by @roydahan in a code review meeting.

**Pull request merged**